package.json

```diff
@@ -6,7 +6,7 @@
     "node": ">=8.0.0"
   },
   "scripts": {
-    "start": "./node_modules/.bin/ts-node -r tsconfig-paths/register src/main/server.ts",
```

Comment:
I'm bootstrapping paths programatically because the `-r` flag is not passed to forked processes. It probably can be done by settings `execArgv` on the `fork` call, but I thought that configuring this in code is cleaner in the end.